cv32e40p/tests/programs/custom/pulp_hardware_loop_interrupt_test/pulp_hardware_loop_interrupt_test.c

```diff
@@ -0,0 +1,251 @@
+#include <stdio.h>
```

Comment:
Please add a license header.

Reply:
This is coming from interrupt_test where there are no headers.